hedera-node/src/main/java/com/hedera/services/context/SideEffectsTracker.java

```diff
@@ -61,9 +62,11 @@
 public class SideEffectsTracker {
 	private static final long INAPPLICABLE_NEW_SUPPLY = -1;
 	private static final int MAX_TOKENS_TOUCHED = 1_000;
+	private static final int BALANCE_CHANGES_LENGTH = 1024;
```

Comment:
I may have gone a bit overboard suggesting 1024...even 64 is plenty, since we limit total balance changes to 20 😁

Reply:
Ah 20 is the limit ! 😄  Used 64 as the initial size.